sigma/backends/loki/loki.py

```diff
@@ -561,19 +561,20 @@ def find_longest_common_string_line_filter(
             return None
         matcher = None
         match = None
-        value = None
+        value: Optional[str] = None
         # Finding the longest common substring of a list of strings, by repeatedly
         # calling SequenceMatcher's find_longest_match. The 1st candidate is cached
         # in the 2nd sequence (b), then following candidates are set as the 1st
         # sequence (a). The longest match between a and b is then found, each time
         # reducing the search region of b based on the previous match.
         # See: https://docs.python.org/3/library/difflib.html#difflib.SequenceMatcher
         for cand in candidates:
-            if matcher is None:
+            if cand is None:
+                return None  # prior check should have caught this, but double checking
+            if matcher is None or value is None:

```

Comment:
Same as above.
```suggestion
            if not matcher or not value:
```

Reply:
The `matcher` and `value` objects are explicitly set to `None` in the previous  lines (562-563) - this is intended to catch those. The addition of `value is None` was simply to stop irrelevant mypy type errors.